quickstart.md

```diff
@@ -63,9 +63,9 @@ with the desired [release tag](https://github.com/confidential-containers/operat
 kubectl apply -k github.com/confidential-containers/operator/config/release?ref=<RELEASE_VERSION>
 ```
 
-For example, to deploy the `v0.8.0` release run:
+For example, to deploy the `v0.9.0` release run:
```

Comment:
Should we update these versions to `v0.10.0` as that's the lowest version that matches these instructions with the `kata-qemu-coco-dev` runtimeclass?

Reply:
Updated